src/components/sumamryStep/SummaryStep.tsx

```diff
@@ -0,0 +1,64 @@
+import { Typography, useTranslation } from '@okp4/ui'
+import type { UseTranslationResponse, DeepReadonly } from '@okp4/ui'
+import type { Metadata } from '../metadataStep/MetadataStep'
+
+type SummaryStepProps = {
+  readonly metadata?: Metadata
+}
+
+// eslint-disable-next-line max-lines-per-function
+export const SummaryStep = ({ metadata }: DeepReadonly<SummaryStepProps>): JSX.Element => {
+  const { t }: UseTranslationResponse = useTranslation()
+
+  return (
+    <div className="okp4-summary-step-main">
+      <Typography as="div" fontSize="small">
+        <div className="okp4-dataspace-step-description">
+          {t('stepper:dataset-deposit:steps:summary:description')}
+        </div>
+      </Typography>
```

Comment:
Small refactoring but it doesn't make sense for me to have two div.

```suggestion
      <div className="okp4-dataspace-step-description">
        <Typography fontSize="small">
          {t('stepper:dataset-deposit:steps:summary:description')}
        </Typography>
      </div>
```